config/api.yml

```diff
@@ -540,6 +540,19 @@
         :identifier: instance_guest_restart
       - :name: reset
         :identifier: instance_reset
+  :notifications:
+    :description: User's past notifications
+    :options:
+    - :collection
+    - :arbitrary_resource_path
```

Comment:
It is here only because of routing. It is temporary until we have another option for this case.

I wanted to create NotificationController < BaseController. Without the `arbitrary_resource_path` we route /api/notifications to the BaseController. I guess we will need some kind of options to tell, whether we want to route to specific OO controller. As some of the endpoints will remain totally generic.

@imtayadeway do you plan to introduce new option to make us route to OO controllers?